release-notes/current.yaml

```diff
@@ -30,6 +30,7 @@ bug fixes: |
   Fix translating backendSettings for extAuth
   Fix an issue that stats compressor was not working.
   Added support for BackendTLSPolicy parsing in Standalone mode.
+  Retrigger reconciliation when backendRef of type ServiceImport is updated.
```

Comment:
or when EndpointSlice(s) for a ServiceImport are updated.

Reply:
because `validateEndpointSliceForReconcile()` ends up calling `isRouteReferencingBackend()` which uses the same indexer setup in `backendHTTPRouteIndexFunc()`